src/list.js

```diff
@@ -0,0 +1,13 @@
+export function toggleCompleteTask(className) { /* eslint-disable-line no-unused-vars */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.